src/test/unit/symbol-store.test.js

```diff
@@ -97,6 +100,18 @@ describe('SymbolStore', function() {
       name: 'last symbol',
       functionOffset: 0,
     });
+
+    const libWithEmptyBreakpadId = {
```

Comment:
This seems like it should be a separate `it` block, that way if it fails we will have a good error message as to why it failed. Right now it feels a bit implicit.

Reply:
I'm having trouble with spurious "timeouts" if I do this so I'm going to land it as-is.